### PR TITLE
WhatsApp Provisioning API DEVX-4915

### PR DESCRIPTION
**Pull request by niati**
## Description

[Review App](https://nexmo-developer-pr-3539.herokuapp.com/messages/whatsapp-provisioning/overview)

### CODE REVIEW

File: _documentation/en/messages/whatsapp-provisioning/overview.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+meta_title: Provision a WhatsApp deployment and manage a WhatsApp Business profile with the WhatsApp Provisioning API. 
+description: The WhatsApp Provisioning API allows you to provision a WhatsApp deployment, perform one time password (OTP) verification, and manage a WhatsApp Business profile. 
+navigation_weight: 1
+---
+
+# WhatsApp Provisioning API Overview
+
+The WhatsApp Provisioning API enables you to deploy a WhatsApp cluster, perform one time password (OTP) verification, and update profile information for a WhatsApp business account.
+
+The first thing you must do is [add a phone number](/add-phone-number) in your WhatsApp Business account dashboard.
```
Comment (slorello89) [inline _documentation/en/messages/whatsapp-provisioning/overview.md:12]: the link here is broken think we're looking for `messages/whatsapp-provisioning/add-phone-number`

File: _documentation/en/messages/whatsapp-provisioning/overview.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Once you've added a phone number, you may [provision your WhatsApp deployment](/provision-deployment).
```
Comment (slorello89) [inline _documentation/en/messages/whatsapp-provisioning/overview.md:14]: This link isn't working - try `/messages/whatsapp-provisioning/provsion-deployment`

File: _documentation/en/messages/whatsapp-provisioning/overview.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Finally, you may use the [Vonage API Dashboard](https://dashboard.nexmo.com/messages/social-channels) to [manage your WhatsApp Business profile](/manage-profile).
```
Comment (slorello89) [inline _documentation/en/messages/whatsapp-provisioning/overview.md:16]: this link isn't working - think we need `/messages/whatsapp-provisioning/manage-profile`

File: _documentation/en/messages/whatsapp-provisioning/add-phone-number.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+3. Navigate to the top bar and click **Phone Numbers**.
+4. Click **Add Phone Number**.
+5. Enter your phone number and display name. Your customers will see the display name when your messages appear in their WhatsApp app.
+6. Click **Next**. You’ll see a confirmation that your display name has been sent for review.
+7. Click **Done**. Your phone number will have a Name pending review label below **Certificate**. Once it’s approved, you’ll see a **View** button.
+8. Below **Certificate**, click **View**. Download your certificate because you will need to use it when provisioning your WhatsApp cluster.
+
+> **NOTE:** Each time you click the **View** button, the certificate will change. Either download or copy your certificate so you can use it in a subsequent step.
+
+You may now [provision your WhatsApp deployment](/provision-deployment).
```
Comment (slorello89) [inline _documentation/en/messages/whatsapp-provisioning/add-phone-number.md:25]: Link isn't working

File: _documentation/en/messages/whatsapp-provisioning/manage-profile.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+# Manage a WhatsApp Business Profile
+
+Once cluster has been successfully deployed, you may now manage your WhatsApp Business profile via the Vonage API Dashboard or the [WhatsApp Provisioning API](/api/whatsapp-provisioning).
+
+To manage your WhatsApp Business profile from the Vonage API dashboard:
+
+> **Note** Your business profile is displayed when WhatsApp users tap to see your full contact information in the WhatsApp mobile app.
+
+1. Navigate to the [Social Channels page(https://dashboard.nexmo.com/messages/social-channels)] on the Vonage API dashboard.
```
Comment (slorello89) [inline _documentation/en/messages/whatsapp-provisioning/manage-profile.md:16]: improperly coupled square brackets are breaking this link.

File: _documentation/en/messages/whatsapp-provisioning/manage-profile.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    * **About** - The text displayed in your profile's **About** section - max 139 characters. It is a brief description seen by new users whenever they contact your business profile.
+    * **Profile picture** - The photo displayed on your profile. Must be a square JPG or PNG with maximum dimensions of 640 pixels X 640 pixels and a maximum file size of 800 kilobytes.
+    * **Business description** - A longer description of your business - max 256 characters. URLs are not hyperlinked.
+    * **Address** - Your business address - max 256 characters.
+    * **Business category** - Your business' industry selected from a list of options.
+    * **Contact email** - Your business' email - max 128 characters.
+    * **Website** - Your business' websites maximum of 2 websites, max website length: 256. URLs are not hyperlinked.
+4. Once you are finished editing the desired profile fields, click **Save**.
+
+## Reference
```
Comment (slorello89) [inline _documentation/en/messages/whatsapp-provisioning/manage-profile.md:28]: I think we should call out here that we can also manage this programmatically - some of our partners may want to build their own dashboards for customers so they'll be interested in doing it programmatically - not sure we need to do much more than point people at the appropriate method in the API Reference.

**Review by slorello89 — CHANGES_REQUESTED** [review]

File: _documentation/en/messages/whatsapp-provisioning/manage-profile.md (hunk 6 above)
Comment (niati) [reply _documentation/en/messages/whatsapp-provisioning/manage-profile.md:28]: There's a mention of it in the opening. Can't figure out how to link directly to that section. Do you think we need to add more about it?

Comment (slorello89) [reply _documentation/en/messages/whatsapp-provisioning/manage-profile.md:28]: ok - I see what I missed, below you refer to the `External Accounts API Reference`, that's actually the WhatsApp provisioning API you're referring to, I'd call it such - external accounts is a separate API.

Comment (niati) [reply _documentation/en/messages/whatsapp-provisioning/manage-profile.md:28]: Good catch! Made the change. Let me know whenever this is ready to go out of Draft status and I'll change it so you can review/approve/merge.

**Comment by laurenannmarie** [comment]
Should we add a note or tool tip around the OTP methods. E.g some numbers can't receive calls from the US so if you have one of those, use SMS. Or if you don't have the number on your device because its a virtual number you can divert to your own mobile device in the dashboard? (Or is this one of these things we do but shouldn't be doing as I appreciate it totally undermines the security aspect of the OTP...

**Comment by laurenannmarie** [comment]
Typo in first line of Managing WhatsApp Profile - "Once cluster has been successfully deployed" -> Once *your*/*the*/*a*?

**Comment by laurenannmarie** [comment]
The only 2 things I think might be missing are: 
1. Something along the lines of  "If you don't have a WhatsApp business account via Facebook and if you aren't already a customer of Vonage using WhatsApp please contact sales" (To stop users struggling through this using the WhatApp Business App or their personal WhatsApp numbers.)
2. The way this is linked to applications. There's one line about creating an application but it doesn't say why you should do this or how you link it (or perhaps I'm missing something)

**Comment by slorello89** [comment]
@niati - I agree with all of Lauren's comments - I think after they are incorporated this is ready to go.

**Comment by niati** [comment]
> The only 2 things I think might be missing are:
> 
> 1. Something along the lines of  "If you don't have a WhatsApp business account via Facebook and if you aren't already a customer of Vonage using WhatsApp please contact sales" (To stop users struggling through this using the WhatApp Business App or their personal WhatsApp numbers.)
> 2. The way this is linked to applications. There's one line about creating an application but it doesn't say why you should do this or how you link it (or perhaps I'm missing something)

Updated addressing the following: 

1. Missing word "the cluster" - See [Manage WhatsApp Business profile](https://nexmo-developer-pr-3539.herokuapp.com/messages/whatsapp-provisioning/manage-profile) page
2. Note about contacting sales if the customer doesn't have a WhatsApp business account - See [Overview](https://nexmo-developer-pr-3539.herokuapp.com/messages/whatsapp-provisioning/overview) page
3. Note clarifying linking WhatsApp account to Vonage account via the dashboard - See [Provision a WhatsApp deployment](https://nexmo-developer-pr-3539.herokuapp.com/messages/whatsapp-provisioning/provsion-deployment) page.

**Comment by laurenannmarie** [comment]
GREAT! Happy with this

**Review by slorello89 — APPROVED** [review]
LGTM 👍

**Pull request merged**